### PR TITLE
Feature/ndpe 42 accessibility testing

### PR DESCRIPTION
**Pull request by visualunderground**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,6 @@ node_modules
 public
 
 assets/scss/nhsuk
+
+tests/pa11y/reports/*.png
```
Comment (ghost) [inline .gitignore]: Would it be better to ignore the entire directory?

**Review by ghost — COMMENTED** [review]
Looks good to me

File: package.json [hunk 2]
```diff
@@ -32,17 +32,18 @@
...
-    "express-hbs": "^1.0.4",
     "gulp-postcss": "^7.0.0",
     "gulp-sass": "^3.1.0",
     "gulp-sourcemaps": "^2.6.1",
     "hbs": "^4.0.1",
     "marked": "^0.3.6",
     "minimist": "^1.2.0",
     "morgan": "~1.8.1",
-    "pa11y-ci": "^1.2.0",
+    "pa11y": "^5.0.0-beta.2",
```
Comment (ghost) [inline package.json]: I think these only need to be devDependencies

Comment (visualunderground) [reply package.json]: You're right - I'll move

File: .gitignore (hunk 1 above)
Comment (visualunderground) [reply .gitignore]: Will do

**Review by stevenadams — APPROVED** [review]
(y)

**Pull request merged**